src/main/java/com/java_academy/gui/Model.java

```diff
@@ -0,0 +1,43 @@
+package com.java_academy.gui;
+
+import com.java_academy.logic.fleet_settings.FleetBuilder;
+import com.java_academy.logic.fleet_settings.ShipSetter;
+import com.java_academy.logic.fleet_settings.shipyard_impl.Randomizer;
+import com.java_academy.logic.model.BoardManager;
+import com.java_academy.logic.model.Cell;
+import com.java_academy.logic.model.Ships;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Bartlomiej Janik
+ * @since 7/31/2017
+ */
+public class Model {
+    private BoardManager board;
+    private Randomizer rand;
+    private Ships ships;
+    private ShipSetter fleetSetter;
+
+
+    public void createRandomizer() {
+        ships = FleetBuilder.getNonLocalizedShips();
+        board = new BoardManager(ships, 10);
+
+        fleetSetter = new ShipSetter(board);
+        rand = new Randomizer(ships, fleetSetter);
+        rand.setFleet();
```

Comment:
Rename method or invoke setFleet() somewhere else because this function does 2 things: creates randomizer and randomizes ships